src/nvapi_drs.cpp

```diff
@@ -174,13 +190,6 @@ extern "C" {
             return Ok(str::format(n, " (", id, "/", name, " = 0x", std::hex, value, ")"));
```

Comment:
… we'd just have to do `itD ? str::fromws(itD->settingNameString) : "Unknown"` here instead of `name`.